### ISSUE TITLE
"no main manifest attribute, in kafkahq.jar" for the latest release

### ISSUE THREAD

**Issue by rockybean** [tchiotludo/kafkahq#31]
When I try to run kafkahq by docker or standalone way, I met the following issue.

"no main manifest attribute, in kafkahq.jar"

As I know, this means the jar misses a Manifest file to specify the Main class.

Any idea?

**Comment by tine2k** [comment]
Seems to me that the build is broken. This is the content of the docker image (0.4.0):
```
/app # ls -la /app
total 2728
drwxrwxr-x    1 root     root          4096 Feb 23 13:57 .
drwxr-xr-x    1 root     root          4096 Feb 23 17:51 ..
-rwxrwxr-x    1 root     root            72 Feb 23 13:54 kafkahq
-rw-rw-r--    1 root     root       2779750 Feb 23 13:57 kafkahq.jar
```
The jar file only has 2.7 MB and does not contain any third-party libs like Micronaut.

**Comment by tchiotludo** [comment]
I confirm !! 

I forgot to update the travis build to follow micronaut shadowJar. 
Build in progress ! 

Thanks for the report :)

**Comment by tchiotludo** [comment]
Fix in 0.4.1 : https://github.com/tchiotludo/kafkahq/releases/tag/0.4.1

Sorry for the failed :face_with_head_bandage:

**Comment by tine2k** [comment]
Thanks for the quick fix! Startup with 0.4.1 works now.
